client/src/legacy/CMSMain.EditForm.js

```diff
@@ -232,7 +232,7 @@ $.entwine('ss', function($){
 
       holder.next().filter('.listbox')[instant ? 'hide' : 'slideUp'](() => {
         holder.removeClass('field--merge-below');
-      });
+      }).css('overflow','visible'); // jquery adding overflow:hidden in hide, this will break listbox display
```

Comment:
if we added line above like this 
holder.next().filter('.listbox').css('overflow','visible')[instant ? 'hide' : 'slideUp'](() 
then it will be overwitten from the 'hide' class again, then it will the same as previous